playground/frontend/lib/modules/examples/components/example_list/expansion_panel_item.dart

```diff
@@ -72,12 +74,35 @@ class ExpansionPanelItem extends StatelessWidget {
                         ? const TextStyle(fontWeight: FontWeight.bold)
                         : const TextStyle(),
                   ),
-                  DescriptionPopoverButton(
-                    parentContext: context,
-                    example: example,
-                    followerAnchor: Alignment.topLeft,
-                    targetAnchor: Alignment.topRight,
-                  ),
+                  Row(
+                    children: [
+                      if (example.isMultiFile)
```

Comment:
@ElessarST may we consider instead having a children list outside this widget and adding the MultifilePopoverButton widget if the `example.isMultiFile` condition passes?  This may improve the readability.  I had to read a few times to understand as it was an if statement inside an array.

Reply:
@damondouglas thanks, I moved actions to the different component for the better readability